src/styles/layout/sidebar.less

```diff
@@ -716,188 +51,31 @@ Sidebar Open
-
-
-
-          /* -----------------------------------------------------------------------------
-          Sidebar Header Sub Label
-          ----------------------------------------------------------------------------- */
-
-          .sidebar-header-sublabel {
-
-            display: flex;
-            margin-top: @base-spacing-unit-screen-medium * 0.1;
-
-            .hostname {
-
-              margin-left: @icon-mini-width + (@base-spacing-unit-screen-medium * 0.1);
-
-            }
-
-            .sidebar-header-sublabel-action {
-
-              margin-left: @base-spacing-unit-screen-medium * 0.1;
-
-            }
-
-          }
-
-        }
-
-
-
-        /* -----------------------------------------------------------------------------
-        Sidebar Content
-        ----------------------------------------------------------------------------- */
-
-        .sidebar-content {
-
-          /* -----------------------------------------------------------------------------
-          Sidbar Menu
-          ----------------------------------------------------------------------------- */
-
-          .sidebar-menu {
-
-            .sidebar-menu-item {
-
-              .sidebar-menu-item-icon {
-
-                margin-right: (@base-spacing-unit-screen-medium * 0.25);
-
-              }
-
-              a {
-
-                padding: (@base-spacing-unit-screen-medium * 0.25) (@grid-container-fluid-narrow-gutter-width-screen-medium * 1.0);
-
-              }
-
-            }
-
-          }
-
-          .sidebar-logo-container {
-
-            display: block;
-
-          }
-
-          .sidebar-menu-item-label {
-            display: flex;
-          }
-
-          .company-name {
-
-            display: inline;
-
-          }
-
-          .sidebar-footer-image {
-
-            margin-bottom: @base-spacing-unit-screen-medium * 0.15;
-
-          }
-
-        }
-
-
-
-        /* -----------------------------------------------------------------------------
-        Sidebar Footer
-        ----------------------------------------------------------------------------- */
-
-        .sidebar-footer {
-
-          .icon-buttons {
-
-            flex-direction: row;
-
-            .button {
-
-              border-top-width: 1px;
-              border-right-width: 1px;
-              border-bottom-width: 0;
-              border-left-width: 0;
-
-              &:last-child {
-
-                border-right-width: 0;
-
-              }
-
-            }
-
-            .user-dropdown {
-
-              .icon {
-                margin-right: (@base-spacing-unit-screen-medium * 0.25);
-              }
-
-              .user-description {
-                display: block;
-              }
-
-              &.dropdown-toggle:after {
-                display: block;
-              }
-
-            }
-
-          }
-
-        }
-
-        & + .page {
-          margin-left: @sidebar-width-screen-medium;
-        }
-
-      }
+      flex-basis: 250px;
```

Comment:
Let's move this width to a variable (I already did this here: https://github.com/dcos/dcos-ui/pull/645)